### PR TITLE
2.10 add-on releases phase 2

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/importurls/CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
+## [8] - 2020-12-15
 ### Changed
 - Update minimum ZAP version to 2.9.0.
```
Comment (kingthorin) [inline addOns/importurls/CHANGELOG.md:8]: 2.10?

Comment (psiinon) [reply addOns/importurls/CHANGELOG.md:8]: Arg, yeah, missed that ;)

Comment (psiinon) [reply addOns/importurls/CHANGELOG.md]: Done

Comment (psiinon) [reply addOns/importurls/CHANGELOG.md]: Actually, dont know why I included that one, have now removed it...

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**